python/fledge/common/microservice_management_client/microservice_management_client.py

```diff
@@ -298,6 +300,28 @@ def update_configuration_item(self, category_name, config_item, category_data):
         response = json.loads(res)
         return response
 
+    def update_service_for_acl_change_security(self, acl, reason):
+        assert reason in ["attachACL", "detachACL", "updateACL"]
+        url = "/fledge/security"
+        payload = {
+            "reason": reason,
+            "argument": acl
```

Comment:
Hi please confirm that you are expecting this payload in your implementation.

Reply:
@deepanshu-yadav In the PR #724 for FOGL-611, I have tested:

curl -v -X PUT 127.0.0.1:41615/fledge/security -d '{"reason" : "attachACL", "argument" : "ACL1"}'